dev/com.ibm.ws.kernel.boot.core/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops

```diff
@@ -153,21 +191,29 @@ option-desc.archive=\
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
 option-key.include=\
 \ \ \ \ --include=value,value,...
-option-desc.include=\
-\tA comma-delimited list of values. The valid values vary depending on \n\
-\tthe action.
+option-desc.javadump.include=\
+\tA comma-delimited list of values. The valid values are: heap, system \n\
+\tthread.
+option-desc.package.include=\
+\tA comma-delimited list of values. The valid values are: all*, usr,   \n\
+\tminify, runnable. If you specify runnable the resulting package will \n\
+\tbe an executable jar that will run the server. The minify option     \n\
```

Comment:
+\tminify, runnable. If you specify runnable **,** the resulting package **is** \n\
+\tbe an executable jar that **runs** the server. The minify option     \n\